model/project_ref.go

```diff
@@ -470,15 +511,38 @@ func FindAllTrackedProjectRefs() ([]ProjectRef, error) {
 		db.NoLimit,
 		&projectRefs,
 	)
-
-	for i := range projectRefs {
-		projectRefs[i].checkDefaultLogger()
+	if err != nil {
+		return nil, err
 	}
 
-	return projectRefs, err
+	return addLoggerAndRepoSettingsToProjects(projectRefs)
+}
+
+func addLoggerAndRepoSettingsToProjects(pRefs []ProjectRef) ([]ProjectRef, error) {
```

Comment:
could this be done in a similar way to FindMergedProjectRef, but as a list of project refs that caches the repo refs?